policyengine_canada/variables/gov/provinces/on/tax/income/credits/oeptc/is_senior_for_oeptc.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_canada.model_api import *
+
+
+class OEPTCSeniorStatus(Enum):
+    SENIOR = "Has at least one senior"
+    NON_SENIOR = "Has no seniors"
+
+
+class is_senior_for_oeptc(Variable):
```

Comment:
Actually could we call this `oeptc_senior_status`? Edge case because this is a bool wrapped as an enum for smart indexing purposes, but I think reserving `is_` variables for true bools will lead to clearer code.